po/zh_CN.po

```diff
@@ -299,15 +299,11 @@ msgid ""
 "require a fencing resource to be explicitly configured, though a "
 "fence_watchdog resource can be configured, to limit use to specific nodes. "
 "If this is set to 0 (the default), the cluster will never assume watchdog-"
-"based self-fencing. If this is set to a negative value, the cluster will use "
-"twice the local value of the `SBD_WATCHDOG_TIMEOUT` environment variable if "
-"that is positive, or otherwise treat this as 0. WARNING: When used, this "
-"timeout must be larger than `SBD_WATCHDOG_TIMEOUT` on all nodes that use "
-"watchdog-based SBD, and Pacemaker will refuse to start on any of those nodes "
-"where this is not true for the local value or SBD is not active. When this "
-"is set to a negative value, `SBD_WATCHDOG_TIMEOUT` must be set to the same "
-"value on all nodes that use SBD, otherwise data corruption or loss could "
-"occur."
+"based self-fencing. Negative values are considered invalid and are "
+"treated as 0. WARNING: When used, this timeout must be larger than "
+"`SBD_WATCHDOG_TIMEOUT` on all nodes that use watchdog-based SBD, and "
+"Pacemaker will refuse to start on any of those nodes where this is not true "
+for the local value or SBD is not active."
```

Comment:
Don't bother updating translations, that gets done for everything for rc1 releases

We don't currently have a formal policy for translation updates, but the idea is that translators can grab rc1 and submit any updates before release

Reply:
> We don't currently have a formal policy for translation updates, but the idea is that translators can grab rc1 and submit any updates before release

For exactly that reason I thought dropping a bit might be easy enough even in a language I don't have an idea of that we might get away without the policy this time ;-) 

dropped